config/locales/en.yml

```diff
@@ -20,4 +20,160 @@
+        first: "Your First PR"
+        grabs: "Up For Grabs"
+        triage: "CodeTriage"
+      welcome:
+        title: "A warm welcome"
+        description_html: "Find a project that has a history and reputation for being welcoming to new contributors in <a href=\"%{url}\">GitHub&apos;s great for new contributors</a> showcase."
+
+  # app/views/home/business.html.erb
+  business:
+    title: "Your business runs on open source software"
+    description: "Once a week, every week. Take time to invest in the open source software you use and love."
+    reason:
+      title: "Your business depends on it"
+      description: "Companies need open source software to run effectively."
+      technology: 
+        title: "Improve the technology you rely on"
+        description: "Direct participation in open source gives your organization a voice in the software you depend on."
+    models:
+      title: "Work better together"
+      description: "Open source development models are at the forefront of software collaboration. Developers who work on open source learn intuitively to improve quality, speed, and transparency at your organization."
+    cost:
+      title: "Reduce overhead"
+      description: "Contributing regularly to open source saves money and developer hours by reducing the custom software you need to maintain."
+    reputation:
+      title: "Attract and retain talent"
+      description: "Employees that work on open source boost their, and their employer's, reputation."
+    plan:
+      title: "Open Source Friday in your organization"
+      description: "Improving your developer culture is too important to be left unplanned."
+      encourage:
+        title: "Set aside time"
+        description: "Each Friday, encourage your managers, employees, and coworkers to spend at least two hours working on the open source projects your organization already uses."
+      policy:
+        title: "Make it a habit"
+        description: "Create clear, internal policies so employees can easily and consistently contribute to open source."
+
+  # app/views/home/maintainers.html.erb
+  maintainers:
+    title: "Build software with help from others"
+    description:  "Every Friday, give your software project some love and make it easier for others to contribute."
+    involve:
+      title: "Help people discover your project"
+      description: "Creating a project is only the beginning. Why not spread the&nbsp;word?"
+      workload:
+        title: "Reduce your workload:"
+        description_html: "Ensure your project's README file details what your project does, its scope and contribution needs, and how to contribute to < a href=\"%{url}\">leverage your community</a>."
+      promote:
+        title: "Promote your work"
+        description_html: "Consider creating a Twitter account, blog posts and website to <a href=\"%{url}\">help people find and follow your project</a>."
```

Comment:
Is there any way to use `link_to` somehow to avoid embedding the `<a href=`? No worries if not, just figured I'd ask.